### PR TITLE
For issue #2646: Usabilla - updated documentation

### PR DESCRIPTION
**Pull request by a-saf**
Updated documentation to reflect Kubernetes ICCS cli commands, updated relevant links to resources.

### CODE REVIEW

File: ibmliberty/index.md [hunk 1]
```diff
@@ -69,39 +69,34 @@ Use one of the free **ibmliberty** images from the {{site.data.keyword.Bluemix_n
 
 **Important:** Before you begin, review the [usage restrictions](#usage) for the **ibmliberty** images.
 
-1.  From the catalog, select **Containers** and choose the **ibmliberty** image to build your container from. If you created your own production-licensed image and deployed it to {{site.data.keyword.Bluemix_notm}}, select this image from the catalog. The container creation page opens.
+1.  From the catalog, select **Containers** --> **IBM Cloud Container Registry** --> **IBM Public Respositories** on the side panel. Search for the **ibmliberty** image to build your container from. If you created your own production-licensed image and deployed it to {{site.data.keyword.Bluemix_notm}}, select this image from the catalog. The container creation page opens.
```
Comment (artberger) [inline ibmliberty/index.md]: Typo in repositories. Also, change arrows to carrots; i.e.
> **Containers** > **IBM Cloud Container Registry** > **IBM Public Repositories**

File: ibmliberty/index.md [hunk 2]
```diff
@@ -69,39 +69,34 @@ Use one of the free **ibmliberty** images from the {{site.data.keyword.Bluemix_n
...
 ## Monitoring the Java heap space usage for a container with the CLI 
 {: #monitor_heap}
 
 
-After you create a container from the **ibmliberty** image, you can list all running processes and review the Java heap usage. The Java heap space is the memory that is available to the Java application during runtime.
+After you create a container from the **ibmliberty** image, you can view metrics on a particular pod and its containers and review the Java heap usage. The Java heap space is the memory that is available to the Java application during runtime.
 {:shortdesc}
 
-1.  List all running processes inside the container.
+1.  See metrics on a particular pod and its containers
```
Comment (artberger) [inline ibmliberty/index.md]: I am wondering if we should have a step before this about getting the pods. Something like:

```
1.  Get the name of the pod that you want to view metrics for.
     kubectl get pods

2.  View metrics for the pod and its containers.
      kubectl top pod <pod_name> --containers
```

Comment (a-saf) [reply ibmliberty/index.md]: Good point, I'll add that step.

File: ibmliberty/index.md (hunk 1 above)
Comment (a-saf) [reply ibmliberty/index.md]: Fixed

**Comment by arthurdm** [comment]
latest version looks good @a-saf 

@artberger - ready for review.

**Pull request merged**